tox.ini

```diff
@@ -0,0 +1,7 @@
+[flake8]
+max-line-length = 130
```

Comment:
looks like this repo is still pointing to an older pre-commit hook version. if you run `pre-commit autoupdate` it'll bump it up to the latest version which you can then commit

Reply:
gotcha done. is there any way to know this ahead of time? I figured it would detect a new version when i initiated the pre-commit originally